pkg/controllers/federatedcluster/clusterstatus.go

```diff
@@ -483,39 +483,32 @@ func shouldCollectClusterStatus(cluster *fedcorev1a1.FederatedCluster, collectIn
 }
 
 func (c *FederatedClusterController) recordClusterStatus(cluster *fedcorev1a1.FederatedCluster, startTime time.Time) {
-	readyReason, joinedReason, offlineReason := clusterutil.GetClusterConditionReasons(&cluster.Status)
```

Comment:
GetClusterConditionReasons. This function can be deleted.

Reply:
Done